server/opts_test.go

```diff
@@ -1142,6 +1142,17 @@ func TestMalformedClusterAddress(t *testing.T) {
 	}
 }
 
+func TestPanic(t *testing.T) {
+	opts, err := ProcessConfigFile("./configs/panic.conf")
```

Comment:
Sorry, I should have mentioned that in the first review. We try now to not add static config files for tests when possible. We have some utilities that will create temporary config files, etc.. So you could rewrite the test this way (and therefore get rid of the new `./configs/panic.conf` file):
```
conf := createConfFile(t, []byte(`port: "this_string_trips_a_panic"`))
defer os.Remove(conf)
opts, err := ProcessConfigFile(conf)
if err == nil {
    t.Fatalf("Expected an error reading config file: got %+v", opts)
} else {
    if !strings.Contains(err.Error(), ":1:0: interface conversion:") {
        t.Fatalf("This was supposed to trip a panic on interface conversion right at the beginning")
    }
}
```

Reply:
incorporated that